scripts/purge-deployments.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euox pipefail
+
+export VALID_NAMESPACES=("hocs-gamma" "hocs-delta" "hocs-qax")
+
+current=$(kubectl config current-context)
+if [[ ${VALID_NAMESPACES[*]} =~ $current ]] ; then
+  echo "Current context is $current"
+else
+  echo "Unauthorised Namespace" >&2
+  exit 1
+fi
+
+echo 'Warning: this action will completely remove all the kubernetes deployments of '${current^^}'. It cannot be undone!'
```

Comment:
I ran this through a linter and it's saying "Double quote to prevent globbing and word splitting." against `${current^^}`. This is because, if this were a proper command, a space in the variable string would be cause the two words to be passed as separate arguments, which is rarely intended. It's less relevant here, but it's useful to know for good practice.

Instead of ending the string to add the variable, consider putting the whole thing in double quotes:
```suggestion
echo "Warning: this action will completely remove all the kubernetes deployments of ${current^^}. It cannot be undone!"
```
(This applies throughout these files.)